src/main/java/com/cleanroommc/groovyscript/compat/mods/prodigytech/AtomicReshaper.java

```diff
@@ -0,0 +1,166 @@
+package com.cleanroommc.groovyscript.compat.mods.prodigytech;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.compat.mods.ModSupport;
+import com.cleanroommc.groovyscript.helper.ingredient.OreDictIngredient;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import lykrast.prodigytech.common.recipe.AtomicReshaperManager;
+import lykrast.prodigytech.common.util.Config;
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RegistryDescription
+public class AtomicReshaper extends VirtualizedRegistry<AtomicReshaperManager.AtomicReshaperRecipe> {
+    @RecipeBuilderDescription(example = {
+        @Example(".input(item('minecraft:gold_ingot')).output(item('minecraft:emerald_block')).primordium(10).time(50)"),
+        @Example(".input(item('minecraft:gold_block')).addOutput(item('minecraft:diamond_block'), 10).addOutput(item('minecraft:carrot'), 3).primordium(7)")
+    })
+    public AtomicReshaper.RecipeBuilder recipeBuilder() {
+        return new AtomicReshaper.RecipeBuilder();
+    }
+
+    @Override
+    public void onReload() {
+        removeScripted().forEach(recipe -> {
+            if (recipe.isOreRecipe())

```

Comment:
in situations where theres a line break needed between the if statement and the body, could you put it in curly braces `{}`

Reply:
to make sure, the project's braces style goes like this, right?
```java
if (x) return y;
if (z_but_with_long_name_so_it_doesnt_fit_on_one_line) {
    return t_but_with_long_name_so_it_doesnt_fit_on_one_line;
)
```